### DISCUSSION TITLE
I can't use QFieldCloud - Custom CRS is not supported

### DISCUSSION THREAD

**Discussion by JuninKorea** [opengisch/QField#2693] [Q&A]
Hello.
I have been using Qfield for a long time but I can't use Qfield with this new update.
I have uploaded my project but it is impossible to download any kind of project including mine.

What is wrong? Anyone knows?
\
![KakaoTalk_20220404_215819374_01](https://user-images.githubusercontent.com/102969856/161549058-3094e0fe-d8b3-46a1-9fa1-e2dedaaaebce.jpg)

**Comment by suricactus** [comment]
Can you please share what is the project CRS you are using and the value you have configured the project area of interest? It seems there is a problem there. Is it a custom one?

**Comment by JuninKorea** [comment]
Thanks for the reply.

Yes. It is customized one.
Basically what I need to use is
EPSG5174 but since it wasn't offered
I customized and it is the project CRS.

If I use different CRS may it work?

I used to use the CRS and it works properly. However after the Qfield Cloud
it isn't working well. 😕

2022년 4월 4일 (월) 오후 10:52, Ivan Ivanov ***@***.***>님이 작성:
